src/hooks/use-user-channels.js

```diff
@@ -0,0 +1,28 @@
+import {useState, useEffect} from 'react'
+
+const useUserChannels = (database, userId) => {
+	const [channels, setChannels] = useState([])
+
+	useEffect(() => {
+		const fetchData = async () => {
+			let res
+			try {
+				res = await database
+					.from('channels')
+					.select(`*`)
+					.eq('user_id', userId)
+				setChannels(res.data)
+			} catch(e) {
+				console.log('error fetching channels', e)
+				throw new Error(e.message)
+			}
+		}
+		fetchData()
+	}, [userId, database])
+
+	return channels
+}
+
+export {
+	useUserChannels
+}
```

Comment:
```suggestion
export default useUserChannels
```

to make it easier to import and like the other ones

Reply:
yes, it is a legacy change, where i was trying to export all `crud` operations for channels here.
But in the end I thought it was not such a good ideas, as other crud (but read) are not hooks (yet?).